tenant-file/portal-app/public/index.html

```diff
@@ -10,6 +10,10 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <script
```

Comment:
Can we set this up with NPM? https://fontawesome.com/how-to-use/on-the-web/setup/using-package-managers